Week1/homework/app.js

```diff
@@ -1,11 +1,139 @@
+      language: "English",
+      author: " J. R. R. Tolkien"
+    },
+    'Harry_Potter_and_The_Chamber_of_Secrets': {
+      title: "Harry Potter and The Chamber of Secrets",
+      language: "English",
+      author: "J. K. Rowling",
+    },
+    'Harry_Potter_and_the_Order_of_the_Phoenix': {
+      title: "Harry Potter and the Order of the Phoenix",
+      language: "English",
+      author: "J. K. Rowling",
+    },
+    'The_Master_and_Margarita': {
+      title: "The Master and Margarita",
+      language: "Russian",
+      author: "Mikhail Bulgakov",
+     
+    },
+    'The_Little_Tragedies': {
+      title: "The Little Tragedies",
+      language: "Russian",
+      author: "Alexander Pushkin",
+    },
+    'Eugene_Onegin': {
+      title: "Eugene Onegin",
+      language: "Russian",
+      author: "Alexander Pushkin",
+    },
+    'The_Cherry_Orchard': {
+      title: "The Cherry Orchard",
+      language: "Russian",
+      author: "Anton Chekhov",
+    },
+    'Evenings_on_a_Farm_Nea_Dikanka': {
+      title: "Evenings on a Farm Nea Dikanka",
+      language: "Russian",
+      author: "Nikolai Gogol",
+    },
+    'Three_Men_in_a_Boat': {
+      title: "Three Men in a Boat",
+      language: "English",
+      author: "Jerome K. Jerome",
+    },
+    'NotreDame_d_Paris': {
+      title: "Notre-Dame de Paris",
+      language: "French",
+      author: "Victor Hugo", 
+    },
+    'The_Shining': {
+      title: "The Shining",
+      language: "English",
+      author: "Stephen King"
+    },
+    'It': {
+      title: "It",
+      language: "English",
+      author: "Stephen King"
+    }
+  };
+  const setImages = {
+    'The_Lord_of_the_Rings': 'images/The_Lord_of_the_Rings.jpg', 
+    'Harry_Potter_and_The_Chamber_of_Secrets': 'images/Harry_Potter_and_The_Chamber_of_Secrets.jpg',
+    'Harry_Potter_and_the_Order_of_the_Phoenix': 'images/Harry_Potter_and_the_Order_of_the_Phoenix.jpg',
+    'The_Master_and_Margarita': 'images/The_Master_and_Margarita.jpg',
+    'The_Little_Tragedies': 'images/The_Little_Tragedies.jpg',
+    'Eugene_Onegin': 'images/Eugene_Onegin.jpg',
+    'The_Cherry_Orchard': 'images/The_Cherry_Orchard.jpg',
+    'Evenings_on_a_Farm_Nea_Dikanka': 'images/Evenings_on_a_Farm_Nea_Dikanka.jpg',
+    'Three_Men_in_a_Boat': 'images/Three_Men_in_a_Boat.jpg',
+    'NotreDame_d_Paris': 'images/NotreDame_d_Paris.jpg',
+    'The_Shining': 'images/The_Shining.jpg',
+    'It': 'images/It.jpg'
+  };
 
-
-  // Replace with your own code
+/*
   console.log(bookTitles);
+*/
+createBookList()
+function createBookList() {
+  let myMainDiv = document.getElementById('main');
+  let myH1 = document.createElement('h1');
+  myMainDiv.appendChild(myH1);
+  let pageTitile = document.createTextNode("My favorite books");
+  myH1.appendChild(pageTitile);
+  let myUl = document.createElement('ul');
+  myMainDiv.appendChild(myUl);
+  for (let i = 0; i < bookTitles.length; i++) {
+    let myLi = document.createElement('li');
```

Comment:
let's assume we created a variable here as the following:
```javascript
const key = bookTitles[i]
```
can you guess what is the value of it and how it can be useful?